server/base/src/main/java/org/apache/accumulo/server/conf/store/impl/PropCacheCaffeineImpl.java

```diff
@@ -113,6 +115,17 @@ public void removeAll() {
     return cache.getIfPresent(propStoreKey);
   }
 
+  /**
+   * This returns a weakly consistent view of the entries in the cache - changes may or may not be
+   * reflected in the view and it is undefined which changes (including eviction) will be visible to
+   * the view.
+   *
+   * @return a map weakly consistent view of the underlying cache entries.
+   */
+  public Map<PropStoreKey<?>,VersionedProperties> asMap() {
+    return Collections.unmodifiableMap(cache.asMap());
+  }
```

Comment:
Would `Map.copyOf(cache.asMap())` be better to provide an immutable snapshot of the current cache?

Reply:
This method has been removed.